server/src/com/thoughtworks/go/config/MergedGoConfig.java

```diff
@@ -101,7 +102,7 @@ public void tryAssembleMergedConfig(GoConfigHolder cruiseConfigHolder,List<Parti
             // save to cache and fire event
             this.saveValidConfigToCacheAndNotifyConfigChangeListeners(newConfigHolder);
         } catch (Exception e) {
-            LOGGER.error(String.format("Failed validation of merged configuration: %s", e));
+            LOGGER.error("Failed validation of merged configuration: {}", e);
```

Comment:
There is no parameter to be substituted here. Don't if it bombs when that happens.
This will use the overridden method
```
 public void error(String msg, Throwable t);
```
Right?